homeassistant/components/binary_sensor/rflink.py

```diff
@@ -0,0 +1,102 @@
+"""
+Support for Rflink binary sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.rflink/
+"""
+import logging
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASSES_SCHEMA, PLATFORM_SCHEMA, BinarySensorDevice)
+from homeassistant.components.rflink import (
+    CONF_ALIASES, CONF_DEVICES, RflinkDevice, cv, vol)
```

Comment:
Import `cv` and `vol` directly, not from the component.

Reply:
Fixed.